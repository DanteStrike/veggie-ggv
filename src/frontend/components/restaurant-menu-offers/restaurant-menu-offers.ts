import AbstractComponent from "../abstract-component/abstract-component";
import {IRestaurantMenuOffers, RestaurantMenuOfferType} from "../../types";

class RestaurantMenuOffers extends AbstractComponent implements IRestaurantMenuOffers {
  constructor(
    private readonly offers: RestaurantMenuOfferType[],
    private readonly classMod: string | null = null
  ) {
    super();
  }

  private getClassMod(): string {
    return this.classMod ? `restaurant-menu__offers-menu--${this.classMod}` : ``;
  }

  protected getTemplate(): string {
    return `
      <ul class="offers-menu restaurant-menu__offers-menu ${this.getClassMod()}">
        ${this.offers.map((offer) => `
          <li class="offers-menu__item clearfix">
            <div class="offers-menu__wrap">
              <p class="offers-menu__offer">
                <span class="offers-menu__offer-title">${offer.title}</span>
                <span class="offers-menu__offer-products">${offer.products.join(`, `)}</span>
              </p>
            </div>
            <span class="offers-menu__offer-price">${offer.price}$</span>
          </li>
        `).join(``)}
      </ul>`;
  }
}

export default RestaurantMenuOffers;
