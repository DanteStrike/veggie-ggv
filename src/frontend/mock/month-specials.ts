import {getRandomNum} from "../utils";


type Currency = '$' | '₽';

interface IMonthSpecial {
  title: string;
  description: string;
  price: {
    value: number;
    currency: Currency;
  }
}

const getMonthSpecial = (): IMonthSpecial => ({
  title: [`Greens fava`, `Greens fava`, `Pea horseradish`, `Soko radicchio`, `Tigernut`, `Yarrow sweet`][getRandomNum(0, 5)],
  description: [`Lorem ipsum dolor sit Pellentesque vel enim a`, `Lorem ipsum dolor sit`, `Pellentesque vel enim a`][getRandomNum(0, 2)],
  price: {
    value: getRandomNum(5, 100),
    currency: ["$", "₽"][getRandomNum(0, 1)] as Currency
  }
});

interface IMonthSpecialsConfig {
  amount: {
    min: number;
    max: number
  }
}

const monthSpecialsConfig: IMonthSpecialsConfig = {
  amount: {
    min: 4,
    max: 8
  }
};

const monthSpecials: IMonthSpecial[] = new Array(getRandomNum(monthSpecialsConfig.amount.min, monthSpecialsConfig.amount.max))
  .fill(``)
  .map(() => getMonthSpecial());


export {monthSpecials};
