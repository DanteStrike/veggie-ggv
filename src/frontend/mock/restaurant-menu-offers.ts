import {RestaurantMenuOfferType} from "../types";
import {RestaurantMenuOffersGroup} from "../enum";

const startersAmount: number = 14;
const startersMock: RestaurantMenuOfferType[] = new Array(startersAmount)
  .fill(``)
  .map((_, index) => ({
    id: index + 1,
    group: RestaurantMenuOffersGroup.STARTERS,
    title: `smoked ricotta terrine (starters ${index + 1})`,
    products: [`ricotta`, `shallots`, `cheddar`, `capers`],
    price: 12
  }));

const mainDishesAmount: number = 10;
const mainDishesMock: RestaurantMenuOfferType[] = new Array(mainDishesAmount)
  .fill(``)
  .map((_, index) => ({
    id: index + startersMock[startersMock.length - 1].id + 1,
    group: RestaurantMenuOffersGroup.MAIN_DISHES,
    title: `pan fried courgette flowers (main dishes ${index + 1})`,
    products: [`courgette flowers`, `goats cheese`, `red onion`, `pine nuts`],
    price: 24
  }));

const dessertsAmount: number = 8;
const dessertsMock: RestaurantMenuOfferType[] = new Array(dessertsAmount)
  .fill(``)
  .map((_, index) => ({
    id: index + mainDishesMock[mainDishesMock.length - 1].id + 1,
    group: RestaurantMenuOffersGroup.DESSERTS,
    title: `samphire fritters with fennel ceviche (desserts ${index + 1})`,
    products: [`fresh samphire`, `plain flour`, `cornflower`, `eggs`, `fennel`],
    price: 20
  }));

const drinksAmount: number = 5;
const drinksMock: RestaurantMenuOfferType[] = new Array(drinksAmount)
  .fill(``)
  .map((_, index) => ({
    id: index + dessertsMock[dessertsMock.length - 1].id + 1,
    group: RestaurantMenuOffersGroup.DRINKS,
    title: `with fennel ceviche (drinks ${index + 1})`,
    products: [`flour`, `cornflower`, `eggs`, `fennel`],
    price: 10
  }));

export const restaurantMenuOffersMock: RestaurantMenuOfferType[] = [...startersMock, ...mainDishesMock, ...dessertsMock, ...drinksMock];
