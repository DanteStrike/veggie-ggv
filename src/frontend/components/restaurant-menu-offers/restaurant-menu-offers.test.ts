import {restaurantMenuOffersMock} from "../../mock/restaurant-menu-offers";
import RestaurantMenuOffers from "./restaurant-menu-offers";

it(`Restaurant Menu Offers component should render correctly`, () => {
  const component = new RestaurantMenuOffers(restaurantMenuOffersMock, `mod`);
  expect(component.getElement()).toMatchSnapshot();
});
