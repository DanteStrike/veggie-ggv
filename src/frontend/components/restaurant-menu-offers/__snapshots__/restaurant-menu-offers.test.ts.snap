// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Restaurant Menu Offers component should render correctly 1`] = `
<ul
  class="offers-menu restaurant-menu__offers-menu restaurant-menu__offers-menu--mod"
>
  
        
          
  <li
    class="offers-menu__item clearfix"
  >
    
            
    <div
      class="offers-menu__wrap"
    >
      
              
      <p
        class="offers-menu__offer"
      >
        
                
        <span
          class="offers-menu__offer-title"
        >
          smoked ricotta terrine (starters 1)
        </span>
        
                
        <span
          class="offers-menu__offer-products"
        >
          ricotta, shallots, cheddar, capers
        </span>
        
              
      </p>
      
            
    </div>
    
            
    <span
      class="offers-menu__offer-price"
    >
      12$
    </span>
    
          
  </li>
  
        
          
  <li
    class="offers-menu__item clearfix"
  >
    
            
    <div
      class="offers-menu__wrap"
    >
      
              
      <p
        class="offers-menu__offer"
      >
        
                
        <span
          class="offers-menu__offer-title"
        >
          smoked ricotta terrine (starters 2)
        </span>
        
                
        <span
          class="offers-menu__offer-products"
        >
          ricotta, shallots, cheddar, capers
        </span>
        
              
      </p>
      
            
    </div>
    
            
    <span
      class="offers-menu__offer-price"
    >
      12$
    </span>
    
          
  </li>
  
        
          
  <li
    class="offers-menu__item clearfix"
  >
    
            
    <div
      class="offers-menu__wrap"
    >
      
              
      <p
        class="offers-menu__offer"
      >
        
                
        <span
          class="offers-menu__offer-title"
        >
          smoked ricotta terrine (starters 3)
        </span>
        
                
        <span
          class="offers-menu__offer-products"
        >
          ricotta, shallots, cheddar, capers
        </span>
        
              
      </p>
      
            
    </div>
    
            
    <span
      class="offers-menu__offer-price"
    >
      12$
    </span>
    
          
  </li>
  
        
          
  <li
    class="offers-menu__item clearfix"
  >
    
            
    <div
      class="offers-menu__wrap"
    >
      
              
      <p
        class="offers-menu__offer"
      >
        
                
        <span
          class="offers-menu__offer-title"
        >
          smoked ricotta terrine (starters 4)
        </span>
        
                
        <span
          class="offers-menu__offer-products"
        >
          ricotta, shallots, cheddar, capers
        </span>
        
              
      </p>
      
            
    </div>
    
            
    <span
      class="offers-menu__offer-price"
    >
      12$
    </span>
    
          
  </li>
  
        
          
  <li
    class="offers-menu__item clearfix"
  >
    
            
    <div
      class="offers-menu__wrap"
    >
      
              
      <p
        class="offers-menu__offer"
      >
        
                
        <span
          class="offers-menu__offer-title"
        >
          smoked ricotta terrine (starters 5)
        </span>
        
                
        <span
          class="offers-menu__offer-products"
        >
          ricotta, shallots, cheddar, capers
        </span>
        
              
      </p>
      
            
    </div>
    
            
    <span
      class="offers-menu__offer-price"
    >
      12$
    </span>
    
          
  </li>
  
        
          
  <li
    class="offers-menu__item clearfix"
  >
    
            
    <div
      class="offers-menu__wrap"
    >
      
              
      <p
        class="offers-menu__offer"
      >
        
                
        <span
          class="offers-menu__offer-title"
        >
          smoked ricotta terrine (starters 6)
        </span>
        
                
        <span
          class="offers-menu__offer-products"
        >
          ricotta, shallots, cheddar, capers
        </span>
        
              
      </p>
      
            
    </div>
    
            
    <span
      class="offers-menu__offer-price"
    >
      12$
    </span>
    
          
  </li>
  
        
          
  <li
    class="offers-menu__item clearfix"
  >
    
            
    <div
      class="offers-menu__wrap"
    >
      
              
      <p
        class="offers-menu__offer"
      >
        
                
        <span
          class="offers-menu__offer-title"
        >
          smoked ricotta terrine (starters 7)
        </span>
        
                
        <span
          class="offers-menu__offer-products"
        >
          ricotta, shallots, cheddar, capers
        </span>
        
              
      </p>
      
            
    </div>
    
            
    <span
      class="offers-menu__offer-price"
    >
      12$
    </span>
    
          
  </li>
  
        
          
  <li
    class="offers-menu__item clearfix"
  >
    
            
    <div
      class="offers-menu__wrap"
    >
      
              
      <p
        class="offers-menu__offer"
      >
        
                
        <span
          class="offers-menu__offer-title"
        >
          smoked ricotta terrine (starters 8)
        </span>
        
                
        <span
          class="offers-menu__offer-products"
        >
          ricotta, shallots, cheddar, capers
        </span>
        
              
      </p>
      
            
    </div>
    
            
    <span
      class="offers-menu__offer-price"
    >
      12$
    </span>
    
          
  </li>
  
        
          
  <li
    class="offers-menu__item clearfix"
  >
    
            
    <div
      class="offers-menu__wrap"
    >
      
              
      <p
        class="offers-menu__offer"
      >
        
                
        <span
          class="offers-menu__offer-title"
        >
          smoked ricotta terrine (starters 9)
        </span>
        
                
        <span
          class="offers-menu__offer-products"
        >
          ricotta, shallots, cheddar, capers
        </span>
        
              
      </p>
      
            
    </div>
    
            
    <span
      class="offers-menu__offer-price"
    >
      12$
    </span>
    
          
  </li>
  
        
          
  <li
    class="offers-menu__item clearfix"
  >
    
            
    <div
      class="offers-menu__wrap"
    >
      
              
      <p
        class="offers-menu__offer"
      >
        
                
        <span
          class="offers-menu__offer-title"
        >
          smoked ricotta terrine (starters 10)
        </span>
        
                
        <span
          class="offers-menu__offer-products"
        >
          ricotta, shallots, cheddar, capers
        </span>
        
              
      </p>
      
            
    </div>
    
            
    <span
      class="offers-menu__offer-price"
    >
      12$
    </span>
    
          
  </li>
  
        
          
  <li
    class="offers-menu__item clearfix"
  >
    
            
    <div
      class="offers-menu__wrap"
    >
      
              
      <p
        class="offers-menu__offer"
      >
        
                
        <span
          class="offers-menu__offer-title"
        >
          smoked ricotta terrine (starters 11)
        </span>
        
                
        <span
          class="offers-menu__offer-products"
        >
          ricotta, shallots, cheddar, capers
        </span>
        
              
      </p>
      
            
    </div>
    
            
    <span
      class="offers-menu__offer-price"
    >
      12$
    </span>
    
          
  </li>
  
        
          
  <li
    class="offers-menu__item clearfix"
  >
    
            
    <div
      class="offers-menu__wrap"
    >
      
              
      <p
        class="offers-menu__offer"
      >
        
                
        <span
          class="offers-menu__offer-title"
        >
          smoked ricotta terrine (starters 12)
        </span>
        
                
        <span
          class="offers-menu__offer-products"
        >
          ricotta, shallots, cheddar, capers
        </span>
        
              
      </p>
      
            
    </div>
    
            
    <span
      class="offers-menu__offer-price"
    >
      12$
    </span>
    
          
  </li>
  
        
          
  <li
    class="offers-menu__item clearfix"
  >
    
            
    <div
      class="offers-menu__wrap"
    >
      
              
      <p
        class="offers-menu__offer"
      >
        
                
        <span
          class="offers-menu__offer-title"
        >
          smoked ricotta terrine (starters 13)
        </span>
        
                
        <span
          class="offers-menu__offer-products"
        >
          ricotta, shallots, cheddar, capers
        </span>
        
              
      </p>
      
            
    </div>
    
            
    <span
      class="offers-menu__offer-price"
    >
      12$
    </span>
    
          
  </li>
  
        
          
  <li
    class="offers-menu__item clearfix"
  >
    
            
    <div
      class="offers-menu__wrap"
    >
      
              
      <p
        class="offers-menu__offer"
      >
        
                
        <span
          class="offers-menu__offer-title"
        >
          smoked ricotta terrine (starters 14)
        </span>
        
                
        <span
          class="offers-menu__offer-products"
        >
          ricotta, shallots, cheddar, capers
        </span>
        
              
      </p>
      
            
    </div>
    
            
    <span
      class="offers-menu__offer-price"
    >
      12$
    </span>
    
          
  </li>
  
        
          
  <li
    class="offers-menu__item clearfix"
  >
    
            
    <div
      class="offers-menu__wrap"
    >
      
              
      <p
        class="offers-menu__offer"
      >
        
                
        <span
          class="offers-menu__offer-title"
        >
          pan fried courgette flowers (main dishes 1)
        </span>
        
                
        <span
          class="offers-menu__offer-products"
        >
          courgette flowers, goats cheese, red onion, pine nuts
        </span>
        
              
      </p>
      
            
    </div>
    
            
    <span
      class="offers-menu__offer-price"
    >
      24$
    </span>
    
          
  </li>
  
        
          
  <li
    class="offers-menu__item clearfix"
  >
    
            
    <div
      class="offers-menu__wrap"
    >
      
              
      <p
        class="offers-menu__offer"
      >
        
                
        <span
          class="offers-menu__offer-title"
        >
          pan fried courgette flowers (main dishes 2)
        </span>
        
                
        <span
          class="offers-menu__offer-products"
        >
          courgette flowers, goats cheese, red onion, pine nuts
        </span>
        
              
      </p>
      
            
    </div>
    
            
    <span
      class="offers-menu__offer-price"
    >
      24$
    </span>
    
          
  </li>
  
        
          
  <li
    class="offers-menu__item clearfix"
  >
    
            
    <div
      class="offers-menu__wrap"
    >
      
              
      <p
        class="offers-menu__offer"
      >
        
                
        <span
          class="offers-menu__offer-title"
        >
          pan fried courgette flowers (main dishes 3)
        </span>
        
                
        <span
          class="offers-menu__offer-products"
        >
          courgette flowers, goats cheese, red onion, pine nuts
        </span>
        
              
      </p>
      
            
    </div>
    
            
    <span
      class="offers-menu__offer-price"
    >
      24$
    </span>
    
          
  </li>
  
        
          
  <li
    class="offers-menu__item clearfix"
  >
    
            
    <div
      class="offers-menu__wrap"
    >
      
              
      <p
        class="offers-menu__offer"
      >
        
                
        <span
          class="offers-menu__offer-title"
        >
          pan fried courgette flowers (main dishes 4)
        </span>
        
                
        <span
          class="offers-menu__offer-products"
        >
          courgette flowers, goats cheese, red onion, pine nuts
        </span>
        
              
      </p>
      
            
    </div>
    
            
    <span
      class="offers-menu__offer-price"
    >
      24$
    </span>
    
          
  </li>
  
        
          
  <li
    class="offers-menu__item clearfix"
  >
    
            
    <div
      class="offers-menu__wrap"
    >
      
              
      <p
        class="offers-menu__offer"
      >
        
                
        <span
          class="offers-menu__offer-title"
        >
          pan fried courgette flowers (main dishes 5)
        </span>
        
                
        <span
          class="offers-menu__offer-products"
        >
          courgette flowers, goats cheese, red onion, pine nuts
        </span>
        
              
      </p>
      
            
    </div>
    
            
    <span
      class="offers-menu__offer-price"
    >
      24$
    </span>
    
          
  </li>
  
        
          
  <li
    class="offers-menu__item clearfix"
  >
    
            
    <div
      class="offers-menu__wrap"
    >
      
              
      <p
        class="offers-menu__offer"
      >
        
                
        <span
          class="offers-menu__offer-title"
        >
          pan fried courgette flowers (main dishes 6)
        </span>
        
                
        <span
          class="offers-menu__offer-products"
        >
          courgette flowers, goats cheese, red onion, pine nuts
        </span>
        
              
      </p>
      
            
    </div>
    
            
    <span
      class="offers-menu__offer-price"
    >
      24$
    </span>
    
          
  </li>
  
        
          
  <li
    class="offers-menu__item clearfix"
  >
    
            
    <div
      class="offers-menu__wrap"
    >
      
              
      <p
        class="offers-menu__offer"
      >
        
                
        <span
          class="offers-menu__offer-title"
        >
          pan fried courgette flowers (main dishes 7)
        </span>
        
                
        <span
          class="offers-menu__offer-products"
        >
          courgette flowers, goats cheese, red onion, pine nuts
        </span>
        
              
      </p>
      
            
    </div>
    
            
    <span
      class="offers-menu__offer-price"
    >
      24$
    </span>
    
          
  </li>
  
        
          
  <li
    class="offers-menu__item clearfix"
  >
    
            
    <div
      class="offers-menu__wrap"
    >
      
              
      <p
        class="offers-menu__offer"
      >
        
                
        <span
          class="offers-menu__offer-title"
        >
          pan fried courgette flowers (main dishes 8)
        </span>
        
                
        <span
          class="offers-menu__offer-products"
        >
          courgette flowers, goats cheese, red onion, pine nuts
        </span>
        
              
      </p>
      
            
    </div>
    
            
    <span
      class="offers-menu__offer-price"
    >
      24$
    </span>
    
          
  </li>
  
        
          
  <li
    class="offers-menu__item clearfix"
  >
    
            
    <div
      class="offers-menu__wrap"
    >
      
              
      <p
        class="offers-menu__offer"
      >
        
                
        <span
          class="offers-menu__offer-title"
        >
          pan fried courgette flowers (main dishes 9)
        </span>
        
                
        <span
          class="offers-menu__offer-products"
        >
          courgette flowers, goats cheese, red onion, pine nuts
        </span>
        
              
      </p>
      
            
    </div>
    
            
    <span
      class="offers-menu__offer-price"
    >
      24$
    </span>
    
          
  </li>
  
        
          
  <li
    class="offers-menu__item clearfix"
  >
    
            
    <div
      class="offers-menu__wrap"
    >
      
              
      <p
        class="offers-menu__offer"
      >
        
                
        <span
          class="offers-menu__offer-title"
        >
          pan fried courgette flowers (main dishes 10)
        </span>
        
                
        <span
          class="offers-menu__offer-products"
        >
          courgette flowers, goats cheese, red onion, pine nuts
        </span>
        
              
      </p>
      
            
    </div>
    
            
    <span
      class="offers-menu__offer-price"
    >
      24$
    </span>
    
          
  </li>
  
        
          
  <li
    class="offers-menu__item clearfix"
  >
    
            
    <div
      class="offers-menu__wrap"
    >
      
              
      <p
        class="offers-menu__offer"
      >
        
                
        <span
          class="offers-menu__offer-title"
        >
          samphire fritters with fennel ceviche (desserts 1)
        </span>
        
                
        <span
          class="offers-menu__offer-products"
        >
          fresh samphire, plain flour, cornflower, eggs, fennel
        </span>
        
              
      </p>
      
            
    </div>
    
            
    <span
      class="offers-menu__offer-price"
    >
      20$
    </span>
    
          
  </li>
  
        
          
  <li
    class="offers-menu__item clearfix"
  >
    
            
    <div
      class="offers-menu__wrap"
    >
      
              
      <p
        class="offers-menu__offer"
      >
        
                
        <span
          class="offers-menu__offer-title"
        >
          samphire fritters with fennel ceviche (desserts 2)
        </span>
        
                
        <span
          class="offers-menu__offer-products"
        >
          fresh samphire, plain flour, cornflower, eggs, fennel
        </span>
        
              
      </p>
      
            
    </div>
    
            
    <span
      class="offers-menu__offer-price"
    >
      20$
    </span>
    
          
  </li>
  
        
          
  <li
    class="offers-menu__item clearfix"
  >
    
            
    <div
      class="offers-menu__wrap"
    >
      
              
      <p
        class="offers-menu__offer"
      >
        
                
        <span
          class="offers-menu__offer-title"
        >
          samphire fritters with fennel ceviche (desserts 3)
        </span>
        
                
        <span
          class="offers-menu__offer-products"
        >
          fresh samphire, plain flour, cornflower, eggs, fennel
        </span>
        
              
      </p>
      
            
    </div>
    
            
    <span
      class="offers-menu__offer-price"
    >
      20$
    </span>
    
          
  </li>
  
        
          
  <li
    class="offers-menu__item clearfix"
  >
    
            
    <div
      class="offers-menu__wrap"
    >
      
              
      <p
        class="offers-menu__offer"
      >
        
                
        <span
          class="offers-menu__offer-title"
        >
          samphire fritters with fennel ceviche (desserts 4)
        </span>
        
                
        <span
          class="offers-menu__offer-products"
        >
          fresh samphire, plain flour, cornflower, eggs, fennel
        </span>
        
              
      </p>
      
            
    </div>
    
            
    <span
      class="offers-menu__offer-price"
    >
      20$
    </span>
    
          
  </li>
  
        
          
  <li
    class="offers-menu__item clearfix"
  >
    
            
    <div
      class="offers-menu__wrap"
    >
      
              
      <p
        class="offers-menu__offer"
      >
        
                
        <span
          class="offers-menu__offer-title"
        >
          samphire fritters with fennel ceviche (desserts 5)
        </span>
        
                
        <span
          class="offers-menu__offer-products"
        >
          fresh samphire, plain flour, cornflower, eggs, fennel
        </span>
        
              
      </p>
      
            
    </div>
    
            
    <span
      class="offers-menu__offer-price"
    >
      20$
    </span>
    
          
  </li>
  
        
          
  <li
    class="offers-menu__item clearfix"
  >
    
            
    <div
      class="offers-menu__wrap"
    >
      
              
      <p
        class="offers-menu__offer"
      >
        
                
        <span
          class="offers-menu__offer-title"
        >
          samphire fritters with fennel ceviche (desserts 6)
        </span>
        
                
        <span
          class="offers-menu__offer-products"
        >
          fresh samphire, plain flour, cornflower, eggs, fennel
        </span>
        
              
      </p>
      
            
    </div>
    
            
    <span
      class="offers-menu__offer-price"
    >
      20$
    </span>
    
          
  </li>
  
        
          
  <li
    class="offers-menu__item clearfix"
  >
    
            
    <div
      class="offers-menu__wrap"
    >
      
              
      <p
        class="offers-menu__offer"
      >
        
                
        <span
          class="offers-menu__offer-title"
        >
          samphire fritters with fennel ceviche (desserts 7)
        </span>
        
                
        <span
          class="offers-menu__offer-products"
        >
          fresh samphire, plain flour, cornflower, eggs, fennel
        </span>
        
              
      </p>
      
            
    </div>
    
            
    <span
      class="offers-menu__offer-price"
    >
      20$
    </span>
    
          
  </li>
  
        
          
  <li
    class="offers-menu__item clearfix"
  >
    
            
    <div
      class="offers-menu__wrap"
    >
      
              
      <p
        class="offers-menu__offer"
      >
        
                
        <span
          class="offers-menu__offer-title"
        >
          samphire fritters with fennel ceviche (desserts 8)
        </span>
        
                
        <span
          class="offers-menu__offer-products"
        >
          fresh samphire, plain flour, cornflower, eggs, fennel
        </span>
        
              
      </p>
      
            
    </div>
    
            
    <span
      class="offers-menu__offer-price"
    >
      20$
    </span>
    
          
  </li>
  
        
          
  <li
    class="offers-menu__item clearfix"
  >
    
            
    <div
      class="offers-menu__wrap"
    >
      
              
      <p
        class="offers-menu__offer"
      >
        
                
        <span
          class="offers-menu__offer-title"
        >
          with fennel ceviche (drinks 1)
        </span>
        
                
        <span
          class="offers-menu__offer-products"
        >
          flour, cornflower, eggs, fennel
        </span>
        
              
      </p>
      
            
    </div>
    
            
    <span
      class="offers-menu__offer-price"
    >
      10$
    </span>
    
          
  </li>
  
        
          
  <li
    class="offers-menu__item clearfix"
  >
    
            
    <div
      class="offers-menu__wrap"
    >
      
              
      <p
        class="offers-menu__offer"
      >
        
                
        <span
          class="offers-menu__offer-title"
        >
          with fennel ceviche (drinks 2)
        </span>
        
                
        <span
          class="offers-menu__offer-products"
        >
          flour, cornflower, eggs, fennel
        </span>
        
              
      </p>
      
            
    </div>
    
            
    <span
      class="offers-menu__offer-price"
    >
      10$
    </span>
    
          
  </li>
  
        
          
  <li
    class="offers-menu__item clearfix"
  >
    
            
    <div
      class="offers-menu__wrap"
    >
      
              
      <p
        class="offers-menu__offer"
      >
        
                
        <span
          class="offers-menu__offer-title"
        >
          with fennel ceviche (drinks 3)
        </span>
        
                
        <span
          class="offers-menu__offer-products"
        >
          flour, cornflower, eggs, fennel
        </span>
        
              
      </p>
      
            
    </div>
    
            
    <span
      class="offers-menu__offer-price"
    >
      10$
    </span>
    
          
  </li>
  
        
          
  <li
    class="offers-menu__item clearfix"
  >
    
            
    <div
      class="offers-menu__wrap"
    >
      
              
      <p
        class="offers-menu__offer"
      >
        
                
        <span
          class="offers-menu__offer-title"
        >
          with fennel ceviche (drinks 4)
        </span>
        
                
        <span
          class="offers-menu__offer-products"
        >
          flour, cornflower, eggs, fennel
        </span>
        
              
      </p>
      
            
    </div>
    
            
    <span
      class="offers-menu__offer-price"
    >
      10$
    </span>
    
          
  </li>
  
        
          
  <li
    class="offers-menu__item clearfix"
  >
    
            
    <div
      class="offers-menu__wrap"
    >
      
              
      <p
        class="offers-menu__offer"
      >
        
                
        <span
          class="offers-menu__offer-title"
        >
          with fennel ceviche (drinks 5)
        </span>
        
                
        <span
          class="offers-menu__offer-products"
        >
          flour, cornflower, eggs, fennel
        </span>
        
              
      </p>
      
            
    </div>
    
            
    <span
      class="offers-menu__offer-price"
    >
      10$
    </span>
    
          
  </li>
  
        
      
</ul>
`;
